docs/sources/administration/service-accounts/create-service-account.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Create a service account in Grafana
+menuTitle: Create a service account
+aliases: [docs/sources/administration/service-accounts/add-service-account-token.md]
+description: 'How to create a service account in Grafana'
+weight: 50
+keywords:
+  - Service accounts
+---
+
+# Create a service account in Grafana
+
+You can create a service account in the service account tab of the organization. To manage your computed workload with a user that is not tied to a user specifically.
```

Comment:
Remove the first sentence because it'll be covered in the task steps. Add more of an intro here about what a service account is. "A service account is a ..... that enables you to ...'. For more information about service accounts, refer to xxx."